src/boss/cloudvm/gcp_worker.go

```diff
@@ -64,6 +68,15 @@ func NewGcpWorkerPool() *WorkerPool {
 	}
 	fmt.Printf("Instance: %s\n", instance)
 
+	fmt.Printf("STEP 2a: prepare snapshot with GCS lambda store config\n")
+	if err := createGcsTemplate(); err != nil {
+		panic(fmt.Errorf("failed to create GCS template.json: %v", err))
+	}
+
+	// Brief pause to ensure filesystem operations are fully committed before snapshot
```

Comment:
What if the change is buffered for longer that 1s?  Can we force it to disk with a sync?  https://pkg.go.dev/os#File.Sync

Reply:
Ah, I see you do sync below.  Was it not working already, without the sleep?  We need to understand this a bit better (I don't trust sleep to robustly solve any problem remaining).